spec/mailers/gsi_mailer_spec.rb

```diff
@@ -1,5 +1,5 @@
-require "spec_helper"
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.